packages/grafana-data/src/dataframe/index.ts

```diff
@@ -6,6 +6,5 @@ export * from './processDataFrame';
 export * from './dimensions';
 export * from './ArrayDataFrame';
 export * from './DataFrameJSON';
-export { StreamingDataFrame, StreamingFrameOptions, StreamingFrameAction } from './StreamingDataFrame';
```

Comment:
Are we breaking backward compatability of `grafana/data` here?

Reply:
What is the policy when it comes to  as alpha/experimental? 
@ryantxu I looked at the guides (https://grafana.com/docs/grafana/latest/developers/plugins/build-a-streaming-data-source-plugin/) and it looks like they only mention `CircularDataFrame`.

Reply:
What is the policy when it comes to using `alpha/experimental` in code? Both in terms of how fast should we promote them to stable and how should we handle breaking changes.
 
btw, @ryantxu I looked at the guides (https://grafana.com/docs/grafana/latest/developers/plugins/build-a-streaming-data-source-plugin/) and it looks like they only mention `CircularDataFrame`.  Wouldn't it make sense to eventually replace it with `StreamingDataFrame` ?

Reply:
> Are we breaking backward compatability of grafana/data here?

maybe -- but on an alpha/experimental class that I have not seen used externally (obviously hard to verify).  Goal in this PR is to move the runtime behavior to core so it is not part of the API surface accidentally.  The actual messages we use and API that should be strict is in DataFrameJSON.